Packs/Slackbot_V2/Integrations/SlackBot/SlackBot.py

```diff
@@ -0,0 +1,920 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+from multiprocessing import Process
+from slack_bolt import App
+from slack_bolt.adapter.socket_mode import SocketModeHandler
+from slack_sdk import WebClient
+import json
+import logging
+import os
+import random
+import requests
+import subprocess
+import sys
+import demisto_client
+import ast
+from demisto_client.demisto_api.rest import ApiException
+
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+# Set default logging
+logging.basicConfig(level=logging.INFO)
+
+# Set integration parameters
+SLACK_BOT_TOKEN = demisto.params().get("bot_token", {}).get("password", "")
+SLACK_APP_TOKEN = demisto.params().get("app_token", {}).get("password", "")
+DEDICATED_CHANNEL = demisto.params().get("dedicated_slack_channel_name")
+DEMISTO_API_KEY = demisto.params().get("api_key", {}).get("password", "")
+DEMISTO_BASE_URL = demisto.params().get("base_url")
+XSIAM_AUTH_ID = demisto.params().get("api_key_id")
+Allowed_roles = demisto.params().get("roles").split(",")
+# TRUST = argToBoolean(demisto.params().get("insecure", False))
+
+# Instantiate the App Class
+try:
+    app = App(token=SLACK_BOT_TOKEN)
+except Exception as e:
```

Comment:
```suggestion
except Exception:
```
If we aren't using the exception info, we can just remove it.